packages/react/src/components/Button/Button.module.css

```diff
@@ -65,6 +65,7 @@
 
   gap: var(--fds-sizing-2);
   min-width: var(--fdsc-button-size);
+  font: var(--fds-typography-paragraph-small);
```

Comment:
You need to also add `font-family: inherit;` after using the CSS `font` shorthand.

It defines font-family and we also want for our users to use their own font-family.